types/babel__helper-validator-identifier/babel__helper-validator-identifier-tests.ts

```diff
@@ -0,0 +1,19 @@
+import {
+    isReservedWord,
+    isStrictReservedWord,
+    isStrictBindOnlyReservedWord,
+    isStrictBindReservedWord,
+    isKeyword,
+    isIdentifierStart,
+    isIdentifierChar,
+    isIdentifierName,
+} from 'babel__helper-validator-identifier';
```

Comment:
```suggestion
} from '@babel/helper-validator-identifier';
```
should use proper namespaced module here